specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Builders.json

```diff
@@ -374,14 +374,14 @@
           ]
         },
         "identity": {
-          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity",
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity",
           "description": "The managed service identities assigned to this resource."
         }
       },
       "description": "Information about the SourceToCloud builder resource.",
       "allOf": [
         {
-          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
```

Comment:
please revert all the changes of ```TrackedResource```, ```ProxyResource``` and ```ManagedServiceIdentity``` . These change will make duplicate schema

Reply:
updated